Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet/App.xaml.cs

```diff
@@ -81,7 +81,13 @@ protected override void OnStart()
                 {
                     Distribute.DisableAutomaticCheckForUpdate();
                 }
+                Task<bool> storageTask = null;
```

Comment:
I think we also have to update Xamarin Native apps (for Android and iOS) and Demo apps.